cajita/src/Cajita_Partitioner.hpp

```diff
@@ -93,6 +107,148 @@ class ManualBlockPartitioner : public BlockPartitioner<NumSpaceDim>
         return _ranks_per_dim;
     }
 
+    /*!
+      \brief Get the owned number of cells of the current MPI rank.
+      \param cart_comm The MPI Cartesian communicator for the partitioning.
+      \param global_cells_per_dim The number of global cells and the global
+      cell offset in each dimension.
+      \param owned_num_cell (Return) The owned number of cells of the current
+      MPI rank in each dimension.
+      \param global_cell_offset (Return) The global cell offset of the current
+      MPI rank in each dimension
+    */
+    void ownedCellInfo(
```

Comment:
Can you add a very simple `tstPartitioner` file that tests these two interfaces for manual and average partitions? Since we moved it here and rely on it in the global grid test I think that would be best. If you don't have time one of us here can help